test/runtime/Policies.go

```diff
@@ -158,7 +158,7 @@ var _ = Describe("RuntimePolicyEnforcement", func() {
 			ExpectEndpointSummary(vm, helpers.Disabled, 0)
 
 			By("Create a new container")
-			vm.ContainerCreate("new", "cilium/demo-httpd", helpers.CiliumDockerNetwork, "-l id.new")
+			vm.ContainerCreate("new", "docker.io/cilium/demo-httpd:latest", helpers.CiliumDockerNetwork, "-l id.new")
```

Comment:
now that you change this one, could you use `helpers.HttpdImage`

Reply:
Done